### PR TITLE
OSDOCS-1652: Adding note about changes from TP

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1652

Preview: https://osdocs-1652-tp-changes--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-nodes-descheduler-ga

### CODE REVIEW

**Review by codyhoag — APPROVED** [review]
LGTM!

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -186,14 +186,19 @@ If a machine config pool (MCP) is in a `degraded` state, the Machine Config Oper
 === Nodes
 
 [id="ocp-4-7-nodes-descheduler-ga"]
-==== Descheduler and profiles GA
+==== Descheduler is generally available
 
 The descheduler is now generally available. The descheduler provides the ability to evict a running pod so that the pod can be rescheduled onto a more suitable node. You can enable one or more of the following descheduler profiles:
 
 * `AffinityAndTaints`: evicts pods that violate basic pod and node affinity and node taints.
```
Comment (kasturinarra) [inline release_notes/ocp-4-7-release-notes.adoc:193]: @bergerhoffer should we say "evicts pods that violate basic pod node affinity & node taints" ? existing one has two 'and's in the sentence so asking.

**Comment by kasturinarra** [comment]
@bergerhoffer one minor comment, rest looks good to me, thanks !!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-7-release-notes.adoc]: @kasturinarra As I mentioned in the other PR, I reworded this. Let me know if it looks better to you now or not.

Comment (kasturinarra) [reply release_notes/ocp-4-7-release-notes.adoc]: @bergerhoffer Thanks, looks good to me now.

**Pull request merged**